### PR TITLE
(slate) Expand selection to full text of link

### PR DESCRIPTION
**Pull request by tiberiuichim**
Fixes the following bug:

- as editor
- go to https://6.demo.plone.org/edit page or a with text with links inside
- select the links block under the "Find out more about Plone", double-click on the "features" word, part of the link "The features of Plone"
- change the URL of the link
- actual behavior (that is considered a bug): instead of only updating the URL of the link, the text of the link also gets changed
- behavior you would expect: in order to change the URL of a link it is not required to select the whole text of the link but only a part

### CODE REVIEW

**Comment by tiberiuichim** [comment]
It needs cypress tests (or at least mending the current cypress tests). I can't run Cypress on my setup, unfortunately.

File: packages/volto-slate/src/editor/plugins/Link/index.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { defineMessages } from 'react-intl'; // , defineMessages
 import { ReactEditor, useSlate } from 'slate-react';
+import { Editor, Transforms, Node } from 'slate'; // Range,
 import { useSelector, useDispatch } from 'react-redux';
 import AddLinkForm from '@plone/volto/components/manage/AnchorPlugin/components/LinkButton/AddLinkForm';
 import {
   _insertElement,
-  _unwrapElement,
+  // _unwrapElement,
```
Comment (robgietema) [inline packages/volto-slate/src/editor/plugins/Link/index.js:9]: Remove this comment

**Review by robgietema — APPROVED** [review]

**Comment by tiberiuichim** [comment]
Don't merge yet, working on tests and fixes

**Comment by tiberiuichim** [comment]
https://user-images.githubusercontent.com/141568/200067861-bb1e62ee-5bcb-4c6f-aa28-c7a3d4ca314f.mp4

**Comment by tiberiuichim** [comment]
@sneridagh I think it's good to go

**Pull request merged**